apps/foundation-demo/src/app/misc/misc.module.ts

```diff
@@ -3,18 +3,29 @@ import { CommonModule } from '@angular/common';
 
 import { NotFoundComponent } from './not-found/not-found.component';
 import { HelpBoxComponent } from './help-box/help-box.component';
+import { HeaderComponent } from './header/components/header/header.component';
+import { FoundationMiscStateModule } from './misc-state.module';
+import { SidebarItemComponent } from './header/components/sidebar-item/sidebar-item.component';
+import { SidebarComponent } from './header/components/sidebar/sidebar.component';
 
 @NgModule({
   imports: [
-    CommonModule
+    CommonModule,
+    FoundationMiscStateModule
   ],
   declarations: [
     NotFoundComponent,
-    HelpBoxComponent
+    HelpBoxComponent,
+    HeaderComponent,
+    SidebarComponent,
```

Comment:
Let's move everything "Sidebar" related into it's own module rather than a part of a more general "misc" module.

Reply:
done